### PR TITLE
Update Service Mapbox Vector Tiles

### PR DESCRIPTION
**Pull request by davidoesch**
Updated according to https://jira.swisstopo.ch/browse/KOGIS_PRJ-1190?focusedCommentId=134160&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-134160

@ltclm  Can you please double check if it corresponds with your structure ?

### CODE REVIEW

File: chsdi/static/doc/source/services/sdiservices.rst [hunk 1]
```diff
@@ -1256,24 +1256,46 @@ A tile request is in the following form:
...
+GetTileSets
+***********
+
+MBTiles for storing  tiled map data in SQLite databases for immediate usage and for efficient transfer. A MBtileSet request is in the following form:
+
+::
+
+    <Scheme>://<ServerName>/tiles/<LayerName>/<version>/<LayerName>.mbtiles
+
+example of one pbf tile:
```
Comment (ltclm) [inline chsdi/static/doc/source/services/sdiservices.rst:1274]: this should be titled with ``example of mbtiles download``

Comment (ltclm) [reply chsdi/static/doc/source/services/sdiservices.rst:1274]: or you should change the path to a pbf path:
```
https://vectortiles.geo.admin.ch/tiles/ch.swisstopo.leichte-basiskarte.vt/v1.0.0/1/1/0.pbf
```

**Review by ltclm — CHANGES_REQUESTED** [review]
thank you @davidoesch 
some small changes. rest looks good to me.

**Comment by rebert** [comment]
@ltclm fine now?

**Review by ltclm — APPROVED** [review]

**Pull request merged**